# Unit 3 - Genetics

1. Describe Mendel’s experimental design.  
   1. Mendel created a (P) generation of true-breeding purple and white-flowered pea plants then purposefully cross-bred them to see how their F1 and F2 generations appeared  
2. Describe what Mendel observed when crossing two contrasting traits.  
   1. The heritability of traits  
3. Define and distinguish between true-breeding organisms, hybrids, the P generation, the F1 generation, and the F2 generation.  
   1. True-breeding: breeding one phenotype for several generations to whittle them down into a homozygous genotype  
   2. P generation: the two true-bred organisms to be crossed  
   3. F1 generation: offspring of P (all should be heterozygous)  
   4. F2 offspring of F1 (phenotype 3:1, genotype 1:2:1)  
4. Define and distinguish between a monohybrid and dihybrid cross	  w
   1. Monohybrid: tracks the inheritance of one trait in a heterozygous individual  
   2. Dihybrid cross: an organism heterozygous for two traits is bred with another one to assess the inheritance of each trait  
   3. Dihybrid cross ratio \= 9:3:3:1  
5. State what percentage of Mendel’s crosses were heterozygous  
   1. 100% of the F1 offspring were heterozygous  
   2. 50% of the F2 offspring were heterozygous  
6. State the five hypotheses of Mendel’s theory.  
   1. Parents don’t transmit traits, they transmit factors which produce those traits (genes inherited)  
   2. Each parent contains 2 copes of a factor (allele), and is either homozygous or heterozygous  
   3. Alternative forms of a factor (allele) lead to alternative traits  
   4. The 2 alleles an individual has do not affect each other  
   5. The presence of an allele doesn’t ensure it will be expressed (only dominant allele is expressed)  
7. State how a testcross reveals the genotype of a dominant trait.  
   1. If a homozygous recessive individual is crossed with a dominant phenotype, we only know the genotype of the latter by the offspring: if half show the recessive phenotype, the latter is heterozygous. If none show the recessive phenotype, it was homozygous dominant  
8. State Mendel’s first law, the law of segregation.  
   1. The two alleles of a trait separate from each other during the formation of gametes, so that half of the gametes will carry on one copy and half will carry the other copy.  
9. Explain how Mendel’s law of segregation describes the inheritance of a single character.  
   1. Each parent contributes one allele to offspring. The phenotype will only show one of those alleles  
10. State Mendel’s second law, the law of independent assortment.  
    1. Genes located on different chromosomes are inherited independently of each other  
11. Explain how Mendel’s law of independent assortment applies to a dihybrid cross.  
    1. A dihybrid cross is a cross between two individuals heterozygous for two genes. It’s used to study the inheritance of two separate traits. According to the lawof independent assortment, its Punnet square will show that the traite are inherited independently.  
12. Describe the genetic relationships between homologous chromosomes.  
    1. Each chromosome of a pair of homologous chromosomes is donated by one parent. While they have the same type of genes in the same locations, they can have different alleles for those same genes.   
13. Explain the role of genetic mutation in evolution.  
    1. Genetic mutations in germ-line cells are the strongest drivers of evolution  
14. List five factors that can disguise Mendelian segregation and explain how they do it.  
    1. Polygenic inheritance/continuous variation: when many genes contribute to a trait  
    2. Pleiotropic effects: one allele affects many phenotypes  
    3. Incomplete dominance: produces heterozygotes unlike either parent  
    4. Environmental effects: phenotype expression changes based on light, temperature, etc.  
    5. epistasis:  interaction between products of two genes in which one modifies the phenotypic expression of the other  
15. Define the chromosome theory of inheritance. Explain the chromosomal basis of the laws of segregation and independent assortment.  
    1. Mendelian traits assort independently because they are determined by genes located on chromosomes that assorted independently in meiosis  
    2. They can do so because gametes are haploid  
16. Explain how and why karyotyping is performed.  
    1. How: Biologists arrest white blood cell division at metaphase, when chromosomes most condensed, with chemicals. Chromosomes stained, paired with homologues, and photographed  
    2. Why: To detect genetic abnormalities in the number of chromosomes  
17. Define nondisjunction, explain how it can occur, and describe aneuploidy.  
    1. Nondisjunction: failure of chromosomes to separate correctly during meiosis I or meiosis II  
    2. Aneuploidy: an abnormal number of chromosomes (usu monosomy and trisomy)  
18. Describe the causes and symptoms of Down syndrome.  
    1. An extra chromosome 21  
    2. More common in older parents  
19. Describe the consequences of abnormal numbers of sex chromosomes.  
    1. Either non-viable fetuses or conditions like Klinefelter syndrome, Turner syndrome, and Down syndrome  
20. Explain how linked genes are inherited differently from non-linked genes.  
    1. Linkage occurs in genes which are closer together on a chromosome. They tend to segregate together  
21. Explain how sex is genetically determined in humans and the significance of the *SRY* gene.  
    1. The SRY gene acts as the genetic "master switch" for male development.

22. List the three questions asked to analyze a human pedigree.  
    1. Is a trait autosomal or sex-linked?  
    2. Is it dominant, recessive, and is the carrier heterozygous or homozygous for the allele?  
    3. Is the trait determined by a single gene or several  
23. Explain how the sickle-cell allele can be adaptive.  
    1. Heterozygous individuals have some resistance against malaria  
24. Explain why human skin coloration is not sufficiently explained by polygenic inheritance.  
    1. There is also epistasis, genes that affect melanin production, and genes that regulate what kind and which pigments are made.  
25. Describe three things geneticists examine in cells obtained by amniocentesis.  
    1. Chromosomal karyotype  
    2. Enzyme activity  
    3. Genetic markers  
26. Define SNP  
    1. Single nucleotide polymorphisms are the spot differences between the version of a gene you have the version another person has. Useful for figuring out which changes lead to disorders.  
27. Explain how the Y chromosome can be used to trace human ancestry.  
    1. Y chromosomes are passed down almost fully intact, and retain and pass down any single nucleotide polymorphisms which make a family line unique.  
28. What is Chargaff’s Rule?  
    1. If guanine \= 25%, cytosine must equal 25%.  
29. Describe the experiments of Griffith, Hershey, and Chase, which supported the idea that DNA was life’s genetic material.  
    1. Griffith: a “transforming factor” was making dead bacteria with capsules intact pathogenic when combined with capsule-less live bacteria  
    2. Hershey and Chase: With DNA and protein labeled with radioactive materials, one could see that it was DNA being injected by viruses into bacterial cells, not protein.  
30. Compare the structures of DNA and RNA.  
    1. DNA/RNA made of a sugar, a phosphate group, and a nitrogenous base  
    2. DNA \= AGCT	RNA \= AGCU  
31. Explain why each nucleotide in Watson and Crick’s DNA structure can form a base pair with only one of the four potential nucleotides.  
    1. Guanine and cytosine are paired by triple hydrogen bonds, while adenine and thymine are paired by double hydrogen bonds  
    2. Adenine and Guanine are purines, large nitrogenous bases  
    3. Cytosine and Thymine are pyrimidines, small nitrogenous bases  
32. Explain how the structure of DNA facilitates its replication.  
    1. DNA is two strands of mirrored nucleotides. Its complementarity makes it possible to copy itself  
33. Show three alternative mechanisms of DNA replication and explain how the Meselson-Stahl experiment confirms one of them.  
    1. Conservative replication: one strand separates and forms two new strands, then disconnects and re-forms into its original while the two copes form their mirror image on their own  
    2. Semi-conservative replication: the most common, confirmed by Meselson-Stahl experiment. One strand separates and each half gains a new complementary side  
    3. Dispersive replication: new and old DNA dispersed among two daughter strands  
34. Describe the four stages of DNA replication.  
    1. Unwinding: the helicase enzyme separates and unwinds parental DNA at the origin of replication site  
    2. Priming the leading strand: an enzyme called primase lays down a short strand of nucleotides called a primer.  
    3. Building the leading strand: DNA polymerase adds to the primer and builds onto the leading strand by adding nucleotides to the 3’ end toward the replication fork.  
       1. DNA polymerase I removes and fills in the RNA primer  
    4. Priming and building the lagging strand: lagging strand assembled discontinuously away from the replication fork. Okazaki fragments linked when DNA polymerase I removes RNA primers.  
    5. Enzyme called DNA ligase joins the ends of the newly synthesized DNA  
35. List the four molecular events that can alter the sequence of DNA and explain how.  
    1. Base substitutions: in germ-line cells, mutations can drive evolution. In somatic cells, mutations can make new allele combinations to the replace the old, then recombination (genes moving to a different chromosome during replication) can drive changes.  
    2. Frameshift mutation: insertion or deletion throws off the reading of a gene  
    3. Changes in chromosome structure: occur during meiosis I. Inversion changes a section’s orientation. Translocation causes a piece of one chromosome to break off and attach to a non-homologous chromosome.  
    4. Transposition: genes move to another place in the genome and its expression or neighboring expressions altered  
36. State the “Central Dogma”  
    1. DNA \-\> RNA \-\> protein  
       1. Genes specify amino acids for a polypeptide  
       2. Polypeptides form proteins  
       3. Proteins determine what a cell will be like  
37. Describe the locations, reactants, and products of transcription  
    1. Occurs in the nucleus (eukaryotes)  
    2. Reactants: helicase, RNA polymerase  
    3. Product: mRNA  
38. Describe the locations, reactants, and products of translation  
    1. Occurs in the cytoplasm  
    2. Reactants: mRNA with codons, ribosome complex with rRNA to stick to mRNA, transfer RNA (tRNA) with anticodons, activating enzymes in cytoplasm to match tRNA with their amino acids  
    3. Product: polypeptide chain  
39. Explain how the “languages” of DNA and RNA are used to produce polypeptides.  
    1. Codons are 3 bases, each of which codes for a specific amino acid  
40. Explain how mRNA is produced using DNA.  
    1. DNA sequence is transcribed onto messenger RNA  
       1. RNA polymerase binds to a special nucleotide sequence called a promoter  
       2. RNA polymerase moves along strand adding complementary nucleotides (CGAU) to mRNA strand  
41. Point out in what chemical direction an mRNA chain is assembled during transcription  
    1. 5’ to 3’  
42. Explain how eukaryotic RNA is processed before leaving the nucleus.  
    1. Exons: segments of DNA transcribed into RNA and translated into protein, makes up 1% to 1.5% of genome  
    2. Introns: extraneous nucleotides, segments of DNA which are cut out  
    3. Results in a mature mRNA transcript  
43. Distinguish between codon and anticodon and indicate which codons do not code for an amino acid.  
    1. Codon: 3-nucleotide sequences on mRNA, each coding for a specific amino acid  
    2. Anticodon: the 3-nucleotide sequence of a tRNA molecule that is complementary to and base pairs with an amino acid-specifying codon in mRNA  
    3. Translational “stop” codons release the polypeptide  
44. Relate the structure of tRNA to its functions in the process of translation.  
    1. tRNA is a string of about 80 nucleotides in a lopped structure with brings aminco acids to ribosomes  
       1. The amino acid attachment site is at the 3’ end  
       2. The anticodon is at the 5’ end  
45. Distinguish between the roles of tRNA and activating enzymes in reading the genetic code.  
    1. Activating enzymes in the cytoplasm match the tRNA with their amino acid, then tRNA attaches to the ribosomal complex and translates the mRNA codon with its anticodon into a polypeptide chain  
46. Describe the structure and function of ribosomes, listing the order in which the A, P and E sites of a ribosome are occupied by an amino acid.   
    1. Ribosomes are the protein-making factories  
       1. Made of 2 subunits which come together to encode polypeptide chains  
       2. Has an mRNA binding site marked by a specific rRNA sequence  
       3. Next to exposed rRNA on ribosomes are three dents labeled A, P, E  
47. Describe the step-by-step process by which amino acids are added to a growing polypeptide chain.  
    1. tRNA brings an amino acid to the A site  
    2. At the P site, peptide bonds form between new amino acid and growing chain  
    3. At the E site, the tRNA is released  
48. Diagram the overall process of transcription and translation.  
    1. Transcription: DNA sequence is transcribed into an RNA sequence with messenger RNA in the nucleus  
    2. Translation: mRNA transcript is used to direct the sequence of amino acids during the synthesis of polypeptides by ribosomes in the cytoplasm  
49. Compare the architecture of prokaryotic and eukaryotic genes.  
    1. Prokaryotic genes are uninterrupted stretches transcribed 3 nucleotides at a time in the cytoplasm  
    2. Eukaryotic genes are located in the nucleus, and are fragmented by exons and introns  
50. Describe the six stages of eukaryotic protein synthesis.  
    1. 1\. Transcription: in nucleus, RNA polymerase transcribes RNA from DNA  
    2. 2\. RNA processing: Introns excised from RNA transcript, remaining exons spliced together, making mRNA  
    3. 3\. mRNA travels to cytoplasm where it binds to ribosomal subunits  
    4. 4\. tRNA molecules become attached to specific amino acids with the help of activating enzymes. Amino acids are brought to ribosome in order specified by mRNA  
    5. 5\. tRNAs bring their amino acids to A site. Peptide bonds form between amino acids at P site. tRNA’s exit ribosome from E site.  
51. Explain the role of promoters in regulating prokaryotic gene expression  
    1. For a gene to be transcribed, RNA polymerase must bind to a promoter: the nucleotide sequence at the end of a gene which allows RNA polymerase to make mRNA  
52. Explain how the *lac* operon works.   
    1. A repressor protein blocks the promoter at the end of the gene until the right inducer (lactose) is present. Lactose binds to the repressor, making it disengage, allowing RNA polymerase to transcribe the gene.  
53. Distinguish between the regulatory action of repressors and activators.  
    1. Repressor: a protein which binds to the DNA blocking the promoter  
    2. Activator: a protein which binds to DNA and makes the promoter more accessible to RNA polymerase  
54. What is gene expression?  
    1. The use of DNA information to direct the production of proteins  
55. Distinguish between the goals of gene expression in prokaryotes and eukaryotes.  
    1. Prokaryotic gene expression is controlled by the quantity and type of nutrients and oxygen in the environment, which alters the types of enzymes present. Some regulatory proteins block the binding of RNA polymerase, some facilitate it.   
    2. Eukaryotic gene expression follows a specifically timed genetic “program” to regulate the body the make sure the correct genes are expressed during development.  
56. Describe how histones associate with DNA in eukaryotic chromosomes.  
    1. The long-term changes in gene expression are achieved by chemical alterations of histones. Methylating histones can make certain regions of DNA more or less available for transcription  
57. List six points where eukaryotic gene expression is regulated.   
    1. Epigenetic modification of chromatin structure (chemically modifying histone proteins to change how DNA is packed/accessed)  
    2. Initiation of transcription: most common, regulating frequency of transcription  
    3. RNA splicing: rate of splicing, alternative splicing to make multiple mRNAs from one gene  
    4. Gene silencing: miRNAs bind mRNA and block translation (miRNA made from inverted, looped RNA to make double-stranded miRNA  
    5. Protein synthesis: regulation, availability of proteins alters rate of expression  
    6. Post translational modification: phosphorylation and other chemical modifications can alter the activity of a protein after it’s produced  
58. Describe the regulatory role of epigenetic modification of DNA and histones in eukaryotes.  
    1. This only modifies gene expression, not encoded information. Methylation can block the accidental transcription by “turning off” genes. Adding a methyl group to cytosine nucleotides, creating 5-methyl-cytosine, can be used in a history core to make genes more or less accessible  
59. Distinguish between basal transcription factors and specific transcription factors.  
    1. Basal transcription factors keep the transcription rate steady, and bring the transcription apparatus and RNA polymerase to a promoter  
    2. Specific transcription factors are numerous and gene-specific. Availability at a certain time and place is highly regulated by multicellular organisms. These bind to the basal transcription factor complex.  
60. Draw a diagram to explain how enhancers regulate gene expression at a distance.  
    1. Enhancer nucleotide sequences bring transcription factors called activators to the promoter, bending/looping DNA to do so, to make mRNA  
61. Define transcription complex.  
    1. Many transcription factors (basal, specific, promoter region, RNA polymerase) combine to regulate genes in response to environmental and developmental signals  
62. Define RNA interference.  
    1. A type of gene silencing in which mRNA transcript is prevented from being translated. It can block mRNA by covering the bases  
       1. Small interfering RNAs (siRNA) bind to mRNA and target its degradation prior to translation  
63. Describe the process and significance of alternative RNA splicing.  
    1. The exons, after introns are removed, can be spliced back together in many ways. This is how 20,000 genes can encode four times as many proteins  
64. Explain how mRNA breakdown, initiation of translation, protein activation, and protein breakdown regulate gene expression.  
65. Describe the significance of miRNA molecules  
    1. miRNA is a type of small interfering RNA (siRNA). It silences genes by binding to RNA, making it unable to be translated.  
66. Explain how gene silencing works.   
    1. Dicer recognizes long, double-stranded RNA and cuts it into miRNA. RISC makes up the miRNA duplex. RISC unwinds miRNA duplex, leaving a single RNA strand which can bind to complementary RNA, blocking translation.  
67. Explain how selective gene expression yields a variety of cell types in multicellular eukaryotes.  
    1. Genes coding for unique functions (e.g., hemoglobin in red blood cells, actin/myosin in muscle cells) are expressed **only** when the cell possesses the correct combination of activators and epigenetic marks.

68. Describe the roles of homeotic genes in development.  
    1. Homeotic genes map an organism’s anatomy. They code for the **transcription factors** that turn on or off the many "downstream" genes responsible for building those structures.  
69. Explain how DNA is packaged into chromosomes.  
    1. DNA is wrapped around histones, like beads on a string. These are wrapped in looped domains around chromatin fibers, which compact into chromosomes.   
70. Explain how a cat’s tortoiseshell coat pattern is formed and why this pattern is only seen in female cats.   
    1. The Y chromosome doesn’t carry a color allele. The multiple color tortoiseshell requires 2 X chromosomes, each carrying a different color allele.  
71. What does a cladogram tell us?  
    1. The changes between species over time based on time of gene change  
72. Define genome, genetic engineering, restriction enzymes and cDNA  
    1. Genome: the genetic information of an organism  
    2. Genetic engineering:the direct manipulation of an organism's genome using biotechnology. It involves the use of techniques to remove, modify, or introduce genetic material (DNA or RNA) from one organism into another to change the characteristics of the recipient.  
    3. Restriction enzymes: enzymes that bind to a specific short DNA sequence, 4-6 nucleotides long)  
    4. cDNA: processed, cytoplasmic mRNA with introns spliced out can interact with reverse transcriptase to make cDNA (complementary DNA)  
73. What is genetics?  
    1. The study of heritable traits/heredity  
74. Describe the four stages of DNA sequencing  
    1. 1\. DNA polymerase enzyme and a primer are added to DNA. As DNA is replicated, fluorescently-tagged nucleotides are added to halt replication. This makes different-length sequences complementary to original DNA  
    2. 2\. Varied-length DNA fragments are separated by gel electrophoresis  
    3. 3\. Computer scan the gel from smallest to largest to determine the sequence, displayed as colored peaks  
    4. 4\. Automated sequencers run, they show the ordered nucleotide sequence.  
75. State the percentage of noncoding genetic material in the human genome and describe the four non-coding types of DNA.  
    1. Introns (24%)  
    2. Structural DNA (too condensed to be transcribed, located near centromeres, telomeres, or ends on chromosome, 20%  
    3. Repeated sequences (10%)  
    4. Transposable elements “transposons” (mobile, parasitic bits of DNA which can replicate and jump to different parts of a chromosome, 45%)  
76. Explain how a restriction enzyme can be used to transfer genes between organisms.  
    1. CRISPR-CAS9 uses this method to chop DNA in an offset way between specific nucleotides, making the ends “sticky”. They can then be joined by DNA ligase in a recombinant DNA molecule  
77. Explain DNA fingerprinting.  
    1. DNA fingerprinting uses unique short-tandem repeats (STRs) to build a genetic profile  
    2. DNA is extracted and the chosen STR region is amplified with polymerase chain reaction (OCR)  
    3. The amplified STR fragments are put in gel and electrified by gel electrophoresis to separate the molecules by size, making a unique profile  
78. Explain how PCR amplifies a DNA sequence.  
    1. Primers define exactly which segments of DNA will be copied, and Taq polymerase enzyme is used because it can withstand the high temps needed for PCR. The three step heat cycle is: hot/denature DNA, cool down to anneal primers to DNA, hot again so Taq polymerase can replicate the sequence  
79. Describe how genetic engineering is being used to produce therapeutic proteins.  
    1. cDNA can be used to make proteins in large quantities via host bacteria to produce anticoagulants for blood clots, FActor VII for improving clotting for hemophilic patients, or producing interferons for stopping viral reproduction  
80. What is a use of cDNA?  
    1. It can be used to determine which cells are expressing specific genes  
81. Describe how a piggyback vaccine is constructed and used.  
    1. A harmless virus like vaccinia Cowpox’s DNA is combined with a harmful virus like herpes and inserted into a cowpox virus  
    2. When the recombinant virus is injected, the immune system creates the correct antibodies  
82. Explain how genetically modified organisms (GMOs) are transforming agriculture, including pest resistance and increased nutritional value.  
    1. Bioengineered pest resistance  
    2. Bioengineered herbicide resistance: DNA particle guns shoot tungsten particles coated in DNA with glyphosate resistance  
    3. Bioengineered nutrition: like golden rice, which has much more iron and vitamin A  
83. Describe the Spemann experiment in 1938 and how the insight by Campbell led to successful animal cloning.   
    1. Spemann proposed that a nucleus form an egg cell could be removed and replaced, but it only worked if the donor nuclei were from early embryos  
    2. Campbell proposed that the egg and nucleus need to be at the same stage of the cell cycle  
84. Explain the role of epigenetics in the genetic reprogramming of cloned cells.   
    1. As mammal eggs/sperm mature, their DNA undergoes reprogramming, also called epigenetics. This alters the adult cells used for cloning  
85. Explain why some stem cells are totipotent and others are not.  
    1. Totipotent cells, embryonic stem cells, have the ability to form any body tissue and even an adult animal because they haven’t differentiated yet  
    2. Yamanaka discovered in 2006 how to make adult stem cells pleuripotent, or able to convert into any cell type (but differentiated)  
86. Diagram the four stages of embryonic stem cell therapy.  
    1. A few days after fertilization, a blastocyst forms  
    2. Embryonic cells taken from inner blastocyst and grown in a tissue culture  
    3. Stem cell culture can be induced to make any tissue cells  
    4. Grown cells can be injected and respond to signals telling it to add to or replace damaged cells  
87. Distinguish between reproductive cloning and therapeutic cloning.  
    1. Reproductive cloning seeks to make an entire cloned organism  
    2. Therapeutic cloning creates an embryo from patient, then it’s destroyed and its embryonic stem cells harvested, cultivated, and injected or transplanted into patient. The patient’s body should recognize it as its own cells.  
88. Describe the gene transfer approach to gene therapy and address the benefits and risks of gene therapy in humans.   
    1. First successful in 1990, a blood disorder caused by a defective enzyme was successfully replaced via bone marrow  
89. Explain how CRISPR is used to edit genes.  
    1. **Cas9 Enzyme (The Scissors):** A DNA-cutting enzyme (a nuclease) that functions like molecular scissors.

    2. **Guide RNA (**{gRNA}**) (The GPS):** A short, engineered RNA molecule that is complementary to the target DNA sequence to be edited. It acts as a guide to bring the Cas9 enzyme to the precise location in the genome

90. State Malthus’ proposition.  
    1. Organisms’ populations tend to increase geometrically, but the ability of humans to increase their food supply increases arithmetically.   
    2. Population sizes remain relatively constant because of death, the basis of natural selection.  
91. Define natural selection  
    1. The differential reproduction of genotypes caused by factors in the environment.  
       1. Leads to evolutionary change  
92. What are the two requirements for natural selection to occur?  
    1. Competition for reproduction  
    2. Heritable traits  
93. Describe the impact of ecological niches on the evolution of Galapagos finch beaks.  
    1. Galapagos finches seemed to have evolved from a single, mainland finch.  
    2. Galapagos had several vacant niches for the birds to fill  
    3. Their beaks, bodies, and habits were uniquely suited to the way they gathered food.  
    4. Their anatomies reflected different selective pressures  
94. Outline a four-step procedure using fossils to test the theory that macroevolution has occurred.  
    1. 1\. Assemble the collection of fossils of a particular group of organisms  
    2. 2\. Date the fossils without knowing what they look like   
    3. 3\. Order them by age  
    4. 4\. Make observations about succession  
95. Differentiate between analogous and homologous structures  
    1. Homologous structures diverged form a common ancestor  
    2. Analogous structures are similar due to a common selection pressure, or “convergent”  
96. Describe molecular clocks like the cytochrome c gene.  
    1. Cytochrome c is an oxidative metabolism protein  
    2. Evolutionary changes accumulate in cytochrome c at a constant rate  
    3. One can plot the time since a pair of organisms presumably diverged by the number of nucleotide differences in cytochrome c.  
97. State the Hardy-Weinberg principle and its assumptions.  
    1. The relative frequencies of alleles in a population don’t change because of Mendelian segregation.   
    2. The sum of frequencies \= 1  
    3. If the frequency of allele A is p and the frequency of allele of allele a is q, then genotypes after one generation are always (p+q) 2 \= p2 \+ 2pq \+q2   
    4. P \+ q \= 1 or p2 \+ 2pq \+ q2 \= 1   
    5. If allele frequencies are not the same now as com compared to past frequencies, the population is undergoing evolutionary change  
98. Discuss five factors that influence Hardy-Weinberg proportions.   
    1. Mutations  
    2. Nonrandom mating   
    3. Genetic Drift: small populations from the founder and bottleneck effect   
    4. Migration  
    5. Selection  
99. Compare stabilizing, disruptive and directional selection and give examples of each.  
    1. Stabilizing selection: phenotypic extremes reduced (intermediate human birth weight)  
    2. Disruptive selection: intermediate types reduced (large-billed and shirt-billed sparrows best suited for only available seeds (large or small, no medium)  
    3. Directional selection: one phenotypic extreme reduced (light moths on dark trees reduced)  
100. Define the biological species concept.  
     1. Defines “species” as groups of actually or potentially interbreeding natural populations which are reproductively isolated from other groups.  
101. Describe five prezygotic and two postzygotic isolating mechanisms.  
     1. Prezygotic isolating mechanisms  
        1. Geographic isolation  
        2. Ecological isolation  
        3. Temporal isolation  
        4. Behavioral isolation  
        5. Mechanical isolation  
        6. Prevention of gamete fusion  
     2. Postzygotic isolation mechanisms  
        1. Hybrid inviability  
        2. Hybrid infertility  
        3. Hybrid breakdown (hybrid offspring sterile)

